library/entity/entity_events/src/main/java/org/quiltmc/qsl/entity_events/mixin/LivingEntityDeathEventMixin.java

```diff
@@ -0,0 +1,27 @@
+package org.quiltmc.qsl.entity_events.mixin;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.damage.DamageSource;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.world.World;
+import org.quiltmc.qsl.entity_events.api.LivingEntityDeathCallback;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(value = { LivingEntity.class, ServerPlayerEntity.class })
```

Comment:
I'd like someone to have a look at this - if it's not acceptable I'm fine with moving it into two separate mixins.

Reply:
This is fine. Multiple targets are something useful